db/schemas/employees.js

```diff
@@ -0,0 +1,60 @@
+db.createCollection('employees', {
+  validator: {
+    $jsonSchema: {
+      bsonType: 'object',
+      required: [
+        'firstName',
+        'lastName',
+        'address1',
+        'city',
+        'country',
+        'wage',
+        'position',
+        'isActive'
+      ],
+      properties: {
+        'firstName': {
+          bsonType: 'string',
+        },
+        'lastName': {
+          bsonType: 'string'
+        },
+        'address1': {
+          bsonType: 'string'
+        },
+        'address2': {
+          bsonType: 'string'
+        },
+        'city': {
+          bsonType: 'string'
+        },
+        'state': {
+          bsonType: 'string'
+        },
+        'country': {
+          bsonType: 'string'
+        },
+        'zipcode': {
+          bsonType: 'string'
```

Comment:
zipcode should be a number

Reply:
I believe its best practice to save zipcodes as a string. please see this stack overflow: https://stackoverflow.com/questions/893454/is-it-a-good-idea-to-use-an-integer-column-for-storing-us-zip-codes-in-a-databas#:~:text=Use%20a%20string%20with%20validation,is%20up%20to%208%20characters).